src/Modules/Events/EventListComponent.js

```diff
@@ -35,26 +36,32 @@ const EventListComponent = props => {
             </select>
           </div>
         </div> */}
-      </div>
-      {Object.keys(events).length === 0 && <h3>No Events Currently Scheduled</h3>}
-      {Object.keys(events).length > 0 && Object.entries(events).map(([date, event]) => (
-        <div key={date} className="row mt-5">
-          <div className="col-md-12">
-            <div className="day-view">
-              <div className="row">
-                <div className="col-md-12">
-                  <span className="day-view-title">{ formatDateDayAndDate(date) }</span>
+        </div>
+      )}
+      {Object.keys(events).length === 0 && (
+        <h3>No Events Currently Scheduled</h3>
+      )}
+      {Object.keys(events).length > 0 &&
+        Object.entries(events).map(([date, event]) => (
+          <div key={date} className="row mt-5">
+            <div className="col-md-12">
+              <div className="day-view">
+                <div className="row">
+                  <div className="col-md-12">
+                    <span className="day-view-title">
+                      {formatDateDayAndDate(date)}
+                    </span>
+                  </div>
+                </div>
+                <div className="row mt-2">
+                  {event.map(event => (
+                    <EventCardComponent key={event.id} event={event} />
+                  ))}
                 </div>
```

Comment:
Not sure exactly how this works, but does this section display dates and then event_dates within a date? From a data perspective, it is possible to have an agency with multiple event_dates/distributions on the same day (e.g. a mobile pantry that operates two distributions on the same day in different counties)

Reply:
The component will show if there are multiple events within a day.

It's "dumb" enough to not care about dates outside of displaying them together.